provider/resources.go

```diff
@@ -600,6 +602,11 @@ func Provider() tfbridge.ProviderInfo {
 			"google_project_access_approval_settings": {
 				Tok: gcpResource(gcpProject, "AccessApprovalSettings"),
 			},
+			// This resource is in the root namespace in the TF provider at the time of writing. The GCP SDK does not
+			// give an obvious namespace choice either. Since an API key authenticates an application, we put it under
+			// the gcpProject module:
+			"google_apikeys_key": {Tok: gcpResource(gcpProject, "ApiKey")},
```

Comment:
Per convo with @viveklak and @lukehoban we feel it's ok to merge as-is. It *is* in the `apikeys` namespace in Google Native, but that's not especially ergonomic since it's the only resource in the namespace.